scripts/build/build.ps1

```diff
@@ -176,6 +176,9 @@ function Invoke-DotNetBuild() {
             /d:sonar.github.oauth=$githubToken `
             /d:sonar.analysis.mode="issues" `
             /d:sonar.scanAllFiles="true" `
+            /d:sonar.analysis.sha1=$githubSha1 `
+            /d:sonar.analysis.prNumber=$githubPullRequest `
+            /d:sonar.branch.name=$branchName `

```

Comment:
Is it necessary to pass a value for _sonar.branch.target_ too?

Reply:
We shall use the `sonar.branch.target` to specify to SQ onto which branch this PR wants to merge. Sadly with the current setup of our builds on CIX we need an update from builders (I am writing the email to build topic while speaking). In the meantime not using it will default to master (which is not always correct but will do for now on).